ui_framework/components/menu/_menu.scss

```diff
@@ -1,26 +1,21 @@
 /**
- * padding-left: 0 and list-style: none rules
- * are meant for ul.kuiMenu and li.kuiMenuItem
- * elements.
- *
- * kuiMenu and kuiMenuItems can classify both
- * nonsemantic divs as well as lists (ul, li).
+ * 1. Allow class to be applied to `ul` and `ol` elements
+ * 2. Allow class to be applied to `li` elements
  */
 .kuiMenu {
-  padding-left: 0;
+  padding-left: 0; /* 1 */
 }
 
 .kuiMenu--contained {
   border: $globalBorderThin;
-  padding-left: 0;
 
   .kuiMenuItem {
     padding: 6px 10px;
   }
 }
 
 .kuiMenuItem {
-  list-style: none;
+  list-style: none; /* 2 */
```

Comment:
Can we scope this comment to the class itself, like in my original comment? As we add more comments, they scale better when the comments are localized like this.